Resources/public/js/alloyeditor/plugins/floatingtoolbar.js

```diff
@@ -36,14 +36,15 @@ YUI.add('ez-alloyeditor-plugin-floatingtoolbar', function (Y) {
     function scrollHandler () {
         var toolbar = document.querySelector(FLOATING_TOOLBAR_SELECTOR),
             editor = findFocusedEditor(),
-            editorRect;
+            editorRect, top;
```

Comment:
CS: please, move `top` to the next line.

Reply:
One more thing, is it firing on every scroll event? If so, maybe it would be a good idea to debounce a bit, for better performance?